eras/dijkstra/src/Cardano/Ledger/Dijkstra/UTxO.hs

```diff
@@ -0,0 +1,51 @@
+{-# LANGUAGE TypeFamilies #-}
+{-# OPTIONS_GHC -Wno-orphans #-}
+
+module Cardano.Ledger.Dijkstra.UTxO () where
+
+import Cardano.Ledger.Alonzo.UTxO (
+  AlonzoEraUTxO (..),
+  AlonzoScriptsNeeded,
+  getAlonzoScriptsHashesNeeded,
+ )
+import Cardano.Ledger.Babbage.UTxO (
+  getBabbageScriptsProvided,
+  getBabbageSpendingDatum,
+  getBabbageSupplementalDataHashes,
+ )
+import Cardano.Ledger.Conway.UTxO (
+  conwayConsumed,
+  conwayProducedValue,
+  getConwayMinFeeTxUtxo,
+  getConwayScriptsNeeded,
+  getConwayWitsVKeyNeeded,
+ )
+import Cardano.Ledger.Dijkstra.Era (DijkstraEra)
+import Cardano.Ledger.Dijkstra.State (EraUTxO (..))
+import Cardano.Ledger.Dijkstra.State.CertState ()
+import Cardano.Ledger.Dijkstra.Tx ()
+import Cardano.Ledger.Mary.UTxO (getConsumedMaryValue)
+
+instance EraUTxO DijkstraEra where
+  type ScriptsNeeded DijkstraEra = AlonzoScriptsNeeded DijkstraEra
+
+  consumed = conwayConsumed
+
+  getConsumedValue = getConsumedMaryValue
+
+  getProducedValue = conwayProducedValue
+
+  getScriptsProvided = getBabbageScriptsProvided
+
+  getScriptsNeeded = getConwayScriptsNeeded
+
+  getScriptsHashesNeeded = getAlonzoScriptsHashesNeeded -- TODO: This also changes for Conway
```

Comment:
I am not really sure what was the original TODO comment about, but I can't see anything that needs to be done for Conway either, so feel free to remove it there as well.
```suggestion
  getScriptsHashesNeeded = getAlonzoScriptsHashesNeeded
```